app/controllers/orga/seasons_controller.rb

```diff
@@ -45,23 +45,13 @@ def destroy
   end
 
   # # switch_phase: enables developers to easily switch between time dependent settings in views
-  # by opening and closing the corresponding links in the nav bar
+  # by showing the corresponding links in the nav bar
   def switch_phase
     return if Rails.env.production?
-      case params[:option]
-      when 'Proposals'
-        Season::PhaseSwitcher.fake_proposals_phase
-      when 'Application'
-        Season::PhaseSwitcher.fake_application_phase
-      when 'CodingSummer'
-        Season::PhaseSwitcher.fake_coding_phase
-      when 'RealTime'
-        Season::PhaseSwitcher.back_to_reality
-      end
-    redirect_to orga_seasons_path, notice: "We time travelled into the #{params[:option]} phase"
+    Season::PhaseSwitcher.destined(params[:phase])
```

Comment:
Well, it's a dev-env-only feature... But anyways, Ruby 2.2 introduced GC for symbols (cc @carpodaster), so as long as you don't do a
```ruby
define_method(string.to_sym)
```
DoS from converting any strings to symbols should not be a problem.

However, if you prefer strings, feel free to switch to them. I just proposed symbols because I personally prefer them over strings. But the whole logic works with strings as good as with symbols (just need to switch `%i()` to `%w()` then).